content/departments/product-engineering/engineering/cloud/devops/managed/creation_process.md

```diff
@@ -21,20 +21,28 @@ For basic operations like accessing an instance for these steps, see [managed in
 
 1. Check out a new branch: `git checkout -b $COMPANY/create-instance`
 1. `./util/create-managed-instance.sh $COMPANY/` and **commit the result**. Make sure that the version exists in [deploy-sourcegraph-docker](https://github.com/sourcegraph/deploy-sourcegraph-docker/tags).
-1. Open and edit `deploy-sourcegraph-managed/$COMPANY/gcp-tfstate/gcp-tfstate.tf` according to the TODO comments within, commit the result.
-1. Ensure you are using the version of Terraform indicated in `.tool-versions` using `tfenv`
-1. In `gcp-tfstate` run `terraform init && terraform apply && git add . && git commit -m 'initialize GCP tfstate bucket'`
-1. Open and edit `infrastructure.tf` according to the TODO comments within and commit the result.
 1. Open and edit `terraform.tfvars` according to the TODO comments within and commit the result.
+
+   | ⚠️ Do not set `enable_alerting` to `true` yet as this will cause false alerts to fire until the MI creation process has been completed! |
+   | --------------------------------------------------------------------------------------------------------------------------------------- |
+
+1. Ensure you are using the version of Terraform indicated in `.tool-versions` using `tfenv`
+1. In `gcp-tfstate` run `terraform init && terraform apply -var-file=../terraform.tfvars && git add . && git commit -m 'initialize GCP tfstate bucket'`
+
+   | ℹ️ You can safely ignore the warnings from `terraform apply`. |
+   | ------------------------------------------------------------- |
+
 1. Open and edit `deploy-sourcegraph-managed/$COMPANY/red/docker-compose/docker-compose.yaml`, increase `gitserver-0`'s `cpus: 8` and `GOMAXPROCS=8` if the instance size is larger than "n1-standard-8".
-1. In `deploy-sourcegraph-managed/$COMPANY` run `./enable-apis.sh`
+
+   | ❓️ what is the purpose of `GOMAXPROCS=8`? Currently it is only added to 1 deployment. |
+   | -------------------------------------------------------------------------------------- |
```

Comment:
@michaellzc any idea?

Reply:
Was this block meant to be added to the commit?

I agree that this is a little odd that we're only setting it on one service manually here. This will also _restrict_ gitserver on machines with more than 8 processors available.

Reply:
GOMAXPROCS is no longer applicable, we can remove this from the doc.

https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/750#discussion_r799585657